RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -76,9 +75,6 @@
     vertexSrc      = 'firstStepGoodPrimaryVertices',
     howToUseMeasurementTracker = 'Never'
 ))
-jetCoreRegionalStepEndcapTrackingRegions = jetCoreRegionalStepTrackingRegions.clone(RegionPSet=dict(
```

Comment:
I miss most of the cleanings in this file: I'm not saying they are not correct, but could you please explain the logic behind? Are all those removed modules/PSets really not needed (or replaced somehow)?

Reply:
> @jeongeun please redo RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py based on a recent IB. As Slava already [pointed out](https://github.com/cms-sw/cmssw/pull/33903#discussion_r645148010) you probably based your code changes on a IB older than CMSSW_12_0_X_2021-05-24-1100, which is when #33531 was merged.

@perrotta Thanks for your comment. I did rebase to move this branch to the recent IB (120X_06-07-2300), but it's strange that nothing different (no conflict) was found. The #33531 update was not applied properly I think..

Reply:
Now it updated to pr #34007.